nipap-cli/nipap

```diff
@@ -74,7 +74,12 @@ if __name__ == '__main__':
         sys.exit(1)
 
     # read configuration
-    cfg = ConfigParser.ConfigParser()
+    cfg_defaults = {
+        'scheme': 'http',
+        'hostname': 'localhost',
```

Comment:
Do we really need a default value for hostname? Users have to define username and password. Is it likely that NIPAP will be running on the same host as the CLI is running on?